pypika/terms.py

```diff
@@ -478,6 +478,15 @@ def get_sql(self, with_alias=False, with_namespace=False, quote_char=None, secon
         return field_sql
 
 
+class Index(Term):
+    def __init__(self, name, alias=None):
+        super(Index, self).__init__(alias)
+        self.name = name
+
+    def get_sql(self, quote_char=None, secondary_quote_char="'", **kwargs):
```

Comment:
The arg `secondary_quote_char` appears to be unused here. Could probably just omit it.

Reply:
True. I removed that one.